### PR TITLE
Updated abbrevation in OJT for better understanding

### PR DESCRIPTION
**Pull request by ksantiagoBAH**
## Description
A few of the abbreviations on the on the job training are being read very literally by screen readers, and are not as clear as they could be. I'd like to consider updating

https://app.zenhub.com/workspaces/vft-59c95ae5fda7577a9b3184f8/issues/department-of-veterans-affairs/va.gov-team/10202

## Testing done
local testing using screen reader

## Acceptance criteria
- [x] Screen readers announce text as natural language, not what is literally on the screen. Screenshots have the requested changes. Please disregard any that are not changed or cannot be changed.

## Definition of done
- [x] Events are logged appropriately
- [x] Documentation has been updated, if applicable
- [x] A link has been provided to the originating GitHub issue (or connected to it via ZenHub)
- [x] No sensitive information (i.e. PII/credentials/internal URLs/etc.) is captured in logging, hardcoded, or specs

### CODE REVIEW

File: src/applications/gi/components/profile/EstimateYourBenefitsForm.jsx [hunk 1]
```diff
@@ -950,21 +950,21 @@ class EstimateYourBenefitsForm extends React.Component {
...
+          { value: '20', label: '20 hours a week' },
+          { value: '18', label: '18 hours a week' },
+          { value: '16', label: '16 hours a week' },
+          { value: '14', label: '14 hours a week' },
+          { value: '12', label: '12 hours a week' },
+          { value: '10', label: '10 hours a week' },
+          { value: '8', label: '8 hours a week' },
+          { value: '6', label: '6 hours a week' },
+          { value: '4', label: '4 hours a week' },
+          { value: '2', label: '2 hours a week' },
```
Comment (shawkey-daniel) [inline src/applications/gi/components/profile/EstimateYourBenefitsForm.jsx:967]: Can we make this "hours per week"?

**Review by shawkey-daniel — CHANGES_REQUESTED** [review]

File: src/applications/gi/components/profile/EstimateYourBenefitsForm.jsx (hunk 1 above)
Comment (ksantiagomorgan6) [reply src/applications/gi/components/profile/EstimateYourBenefitsForm.jsx:967]: Yeah, good catch

**Review by 1Copenut — APPROVED** [review]
LGTM. I pulled the feature branch and verified locally using VoiceOver. I'll spot check in Staging on NVDA and JAWS when ready.

**Review by shawkey-daniel — APPROVED** [review]
LGTM

**Review by delli-gatti-michael — APPROVED** [review]
~looks~ sounds good to me

**Pull request merged**